src/components/404/Error404.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
```

Comment:
please, move all components that are used in Redux to show the route in pages

Reply:
I am sorry, I do not completely understand what you mean here

Reply:
Sorry, I wrote something funny
I meant:
'please, move all components that are used in Router as pages in pages folder'

so, this file is a page, we can see that in App.js, as well as some other component
let's move theirs files in folder /pages where Home.js and MoviePage are